docs/options.md

```diff
@@ -191,6 +193,9 @@ If not listed above, `auto` is the same as `native`.
 [setup-qemu-action]: https://github.com/docker/setup-qemu-action
 [binfmt]: https://hub.docker.com/r/tonistiigi/binfmt
 
+Platform-specific variants also available:<br/>
+ `CIBW_ARCHS_MACOS` | `CIBW_ARCHS_WINDOWS` | `CIBW_ARCHS_LINUX`
+
```

Comment:
Good catch, I forgot that 👍

Reply:
Actually, this was the result of a bad rebase - there are two of these now! 😳

Reply:
I think the "normal" place is here, right before examples, rather than after. I've kept this one.